server/agents/providers/lemonade_provider.py

```diff
@@ -0,0 +1,109 @@
+"""Lemonade runtime provider implementation."""
+
+import sys
+import time
+from pathlib import Path
+import requests
+import instructor
+from openai import AsyncOpenAI
+
+# Add repo root to path for config imports
+repo_root = Path(__file__).parent.parent.parent.parent
+sys.path.insert(0, str(repo_root))
+
+from config.datamodel import LlamaFarmConfig, PromptFormat  # noqa: E402
+from .base import RuntimeProvider
+
+
+class LemonadeProvider(RuntimeProvider):
+    """Lemonade local runtime provider implementation."""
+
+    def get_base_url(self, config: LlamaFarmConfig) -> str:
+        """Get base URL for Lemonade API."""
+        if config.runtime.base_url:
+            return config.runtime.base_url
+
+        port = 11534  # default
```

Comment:
This shou;ld not be hardcoded, but rather moved to server/core/settings.py and referenced here by a settings import